workers/ohsome_quality_analyst/indicators/last_edit/indicator.py

```diff
@@ -23,83 +25,79 @@ def __init__(
+                    "%Y-%m-%d"
+                ),
+                end=latest_ohsome_stamp.strftime("%Y-%m-%d"),
             )
 
-        query_results_contributions = await ohsome_client.query(
+        response = await ohsome_client.query(
             layer=self.layer,
             bpolys=json.dumps(self.bpolys),
             time=self.time_range,
-            endpoint="contributions/latest/centroid",
+            endpoint="contributions/latest/count",
         )
-        query_results_totals = await ohsome_client.query(
+        self.contributions_latest_count = response["result"][0]["value"]
+        response = await ohsome_client.query(
             layer=self.layer,
             bpolys=json.dumps(self.bpolys),
         )
-        if query_results_contributions is None or query_results_totals is None:
-            return False
-        try:
-            self.edited_features = len(query_results_contributions["features"])
-        except KeyError:
-            # no feature has been edited in the time range
-            self.edited_features = 0
-
-        self.total_features = query_results_totals["result"][0]["value"]
-        if self.total_features != 0 and self.total_features is not None:
-            if self.edited_features > self.total_features:
-                # It can happen that features are counted which has been deleted since.
-                self.share_edited_features = 100
-            else:
-                self.share_edited_features = round(
-                    (self.edited_features / self.total_features) * 100
-                )
+        self.element_count = response["result"][0]["value"]
+
         return True
 
     def calculate(self) -> bool:
         logging.info(f"Calculation for indicator: {self.metadata.name}")
 
-        description = Template(self.metadata.result_description).substitute(
-            share=self.share_edited_features, layer_name=self.layer.name
-        )
-        if self.total_features == 0:
+        # It can be that features are counted, but have been deleted since.
+        if self.element_count == 0:
             self.result.label = "undefined"
             self.result.value = None
-            self.result.description = (
-                "Since the OHSOME query returned a count of 0 for this feature "
-                "a quality estimation can not be made for this filter"
-            )
+            self.result.description = "In the AOI no features are present."
             return False
-        elif self.share_edited_features >= self.threshold_yellow:
+
+        if self.contributions_latest_count > self.element_count:
+            self.ratio = 100
+        else:
+            self.ratio = round(
+                (self.contributions_latest_count / self.element_count) * 100
+            )
+
+        description = Template(self.metadata.result_description).substitute(
+            share=self.contributions_latest_count, layer_name=self.layer.name
```

Comment:
Shouldn't the share be `self.ratio`?

Reply:
I changed in here and in metadata.yml to ratio,

Reply:
I meant the value to the parameter `share` should be `self.ratio`, not the name itself ;)

Reply:
Oh. Thats right of course. Than this has been wrong before the changes of this PR.
I fixed this.